docs/pages/desktop-access/introduction.mdx

```diff
@@ -1,44 +1,72 @@
 ---
-title: Desktop Access
-description: Teleport desktop access introduction and resources.
+title: Manage Windows resources
```

Comment:
We still advise using title case in the style guide. If we want to stop using title case (which I agree with, TBH), we should update the style guide and plan to update other guides in the docs. Otherwise we'll just have inconsistency.

Reply:
That's kind of what sent me down changing the style guide path.